README.md

```diff
@@ -1,53 +1,68 @@
 # [<img src="ttlogo.png" width="300" alt="Terratag Logo">](https://terratag.io)
+
 [![ci](https://github.com/env0/terratag/workflows/ci/badge.svg)](https://github.com/env0/terratag/actions?query=workflow%3Aci+branch%3Amaster) [![FOSSA Status](https://app.fossa.com/api/projects/git%2Bgithub.com%2Fenv0%2Fterratag.svg?type=small)](https://app.fossa.com/projects/git%2Bgithub.com%2Fenv0%2Fterratag?ref=badge_small)
 
 > <sub>Terratag is brought to you with&nbsp;❤️&nbsp; by
->[<img src="logo.svg" width="150">](https://env0.com)
+> [<img src="logo.svg" width="150">](https://env0.com)
 > Let your team manage their own environment in AWS, Azure and Google. <br/>
 > Governed by your policies and with complete visibility and cost management.
 
 ## What?
+
 Terratag is a CLI tool allowing for tags or labels to be applied across an entire set of Terraform files. Terratag will apply tags or labels to any AWS, GCP and Azure resources.
 
 ### Terratag in action
+
 ![](https://assets.website-files.com/5dc3f52851595b160ba99670/5f62090d2d532ca35e143133_terratag.gif)
 
 ## Why?
+
 Maintaining tags across your application is hard, especially when done manually. Terratag enables you to easily add tags to your existing IaC and benefit from some cross-resource tag applications you wish you had thought of when you had just started writing your Terraform, saving you tons of time and making future updates easy. [Read more](https://d1.awsstatic.com/whitepapers/aws-tagging-best-practices.pdf) on why tagging is important.
 
 ## How?
+
 ### Prerequisites
-- Terraform 0.11 through 1.3
+
+- Terraform 0.12 through 1.3
```

Comment:
0.11 does not have the required command.
We cannot use tfschema because it's insecure.

0.11 was released ~6 years ago... I guess it's time to remove it.